File: Python/Course.py
class Course :
    # Konstruktor
    def __init__(self, namaMatkul = ""):
        self.__namaMatkul = namaMatkul
    
    # get
    def get_namaMatkul(self) :
        return self.__namaMatkul;
    
    # set
    def set_namaMatkul(self, namaMatkul):
        self.__namaMatkul = namaMatkul
    

File: C++/Course.cpp
#include <iostream>
#include <string>
using namespace std;

class Course{
    // buat variabelnya dlu
    protected:
        string namaMatkul;

    public:
        // konstruktor
        Course(string namaMatkul){
            this->namaMatkul = namaMatkul;
        }
        
        //Getter
        string getMatkul() {
            return this->namaMatkul;
        }
        
        // Setter
        void setnamaMatkul(string namaMatkul) {
            this->namaMatkul = namaMatkul;
        }
        // Destruktor
        ~Course(){
        }
};